buildSrc/src/main/groovy/org/elasticsearch/gradle/vagrant/VagrantCommandTask.groovy

```diff
@@ -29,16 +30,23 @@ import javax.inject.Inject
  * Runs a vagrant command. Pretty much like Exec task but with a nicer output
  * formatter and defaults to `vagrant` as first part of commandLine.
  */
-public class VagrantCommandTask extends LoggedExec {
+public class VagrantCommandTask extends Exec {
```

Comment:
Hum, this should be `LoggedExec` instead. This is a left over from testing changes.